bitcoin/secp256k1_main.py

```diff
@@ -300,9 +291,9 @@ def ecdsa_raw_sign(msg,
         raise Exception("Invalid hash input to ECDSA raw sign.")
     if rawpriv:
         compressed, p = read_privkey(priv)
-        newpriv = secp256k1.PrivateKey(p, raw=True, ctx=ctx)
+        newpriv = coincurve.PrivateKey(p)
     else:
-        newpriv = secp256k1.PrivateKey(priv, raw=False, ctx=ctx)
+        newpriv = coincurve.PrivateKey.from_der(priv)
```

Comment:
We don't yet have a non-raw-privatekey usecase, this seems like a sensible choice (`from_der`) out of the given options.